src/test/common/page/add-children-dob.js

```diff
@@ -22,6 +27,71 @@ class AddChildrenDOB extends SubmittablePage {
   getPageName () {
     return 'add your childrens dates of birth'
   }
+
+  async clickAddAnotherChild () {
+    const addAnotherChildButton = await this.findById('add-another-child')
+    await addAnotherChildButton.click()
+  }
+
+  getDateLastYear () {
```

Comment:
The same method exists in the accessibility tests, we should move it into somewhere common.

Reply:
Yes, I've been thinking about doing some refactoring of date based tests so I'll look at it in another PR.